server/liveblog/posts/posts.py

```diff
@@ -143,7 +143,8 @@ def on_update(self, updates, original):
         post.update(updates)
         self.check_post_permission(post)
         # put the published item from drafts at the top of the timeline
-        if updates.get('post_status') == 'open' and original.get('post_status') == 'draft':
+        if (updates.get('post_status') == 'open' and original.get('post_status') == 'draft'
```

Comment:
The parentheses seem wrong to prevent an unwanted operation priority here.
But anyway, I would simplify it like that:
```python
if updates.get('post_status') == 'open' and original.get('post_status') in ('draft', 'submitted'):
```